airflow/config_templates/config.yml

```
@@ -1222,7 +1222,9 @@
     - name: worker_class
       description: |
         The worker class gunicorn should use. Choices include
-        sync (default), eventlet, gevent
+        sync (default), eventlet, gevent. Note that in case of gevent you might also want to set
```

```suggestion
        sync (default), eventlet, gevent. Note when using gevent you might also want to set the
```

nit, just sounds better to my ear